backup/dependencies.go

```diff
@@ -73,7 +74,7 @@ var (
 	PG_TYPE_OID                 uint32 = 1247
 	PG_USER_MAPPING_OID         uint32 = 1418
 
-	FIRST_NORMAL_OBJECT_ID      uint32 = 16384
```

Comment:
Please, remove all formatting changes that doesn't matter for this patch to decrease conflicts footprint.

Reply:
I've fixed all the formatting issues